alpha/apps/kaltura/modules/extwidget/actions/downloadAction.class.php

```diff
@@ -226,14 +215,9 @@ private function dumpFile($file_path, $file_name, $limit_file_size = 0, $key = n
 		{
 			if(!$directServe)
 				header("Content-Disposition: attachment; filename=\"$file_name\"");
-
-			$mime_type = null;
-			if(!$allowRemote)
-			{
-				$mime_type = kFile::mimeType($file_path);
-			}
-
-			kFileUtils::dumpFile($file_path, $mime_type, null, $limit_file_size, $key, $iv, $fileSize, $allowRemote, $fileExt);
+			
+			$mime_type = kFile::mimeType($file_path);
+			kFileUtils::dumpFile($file_path, $mime_type, null, $limit_file_size, $key, $iv, $fileSize, false, $fileExt);
```

Comment:
if all the references to kFileUtils::dumpFile are now with allowRemote=false maybe worth removing it at all from the function if there is no use for it anymore